gossip/gossip.go

```diff
@@ -103,6 +103,13 @@ var (
 
 func init() {
 	gob.Register(&proto.NodeDescriptor{})
+	gob.Register(&proto.ZoneConfig{})
```

Comment:
Seems like it would be better to move these into util/gob.go (or similar). Strange to confine these to gossip.